bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntry.java

```diff
@@ -32,7 +32,7 @@
  */
 @Public
 @Unstable
-public interface LedgerEntry {
+public interface LedgerEntry extends AutoCloseable {
```

Comment:
Should probably be a different commit, but there is an inconsistency in the data types used in read and write.

WriteHandle is takes ByteBuf and ByteBuffer
ReadHandle/LedgerEntry returns ByteBuf and byte[]

This should be resolved.

Reply:
#750 opened for it.

Reply:
@ivankelly : a good point. it was inconsistent there because I asked @eolivelli to not use `byte[]`. but @eolivelli wants to have a `byte[]` array. that was probably what caused confusion in the initial commit. we can support all - `byte[]`, `ByteBuffer` and `ByteBuf`.